.wav sound file not able to build on MonoGame 3.8

<!--
# Please make sure that the issue is present in the
# develop branch of MonoGame before reporting
#
# You can download the development build installer from:
# http://www.monogame.net/downloads/
-->

<!-- Write your issue below -->

Sound files (*.wav) are not able to build in MonoGame 3.8(develop branch). It raises build errors as the following:
```
You can invoke the tool from this directory using the following commands: 'dotnet tool run mgcb' or 'dotnet mgcb'.
Tool 'dotnet-mgcb' (version '3.7.200-develop') was successfully installed. Entry is added to the manifest file /Users/Ryan/monogame/Pong/Pong.DesktopGL/.config/dotnet-tools.json.
/Users/Ryan/monogame/Pong/Pong.DesktopGL/Content/Sound/1.wav
Failed to open file 1. Ensure the file is a valid audio file and is not DRM protected.
/Users/Ryan/.nuget/packages/monogame.content.builder.task/3.7.200-develop/build/MonoGame.Content.Builder.Task.targets(269,5): error MSB3073: The command "dotnet tool run mgcb /quiet /@:"/Users/Ryan/monogame/Pong/Pong.DesktopGL/Content/Content.mgcb" /platform:DesktopGL /outputDir:"/Users/Ryan/monogame/Pong/Pong.DesktopGL/Content/bin/DesktopGL/Content" /intermediateDir:"/Users/Ryan/monogame/Pong/Pong.DesktopGL/Content/obj/DesktopGL/Content" /workingDir:"/Users/Ryan/monogame/Pong/Pong.DesktopGL/Content/"" exited with code 1. [/Users/Ryan/monogame/Pong/Pong.DesktopGL/Pong.DesktopGL.csproj]
```
The build error: **Failed to open file 1. Ensure the file is a valid audio file and is not DRM protected.**

Test project: https://github.com/ryancheung/Pong


<!-- System stats -->

#### What version of MonoGame does the bug occur on:
- MonoGame 3.8 (develop branch)

#### What operating system are you using:
- MacOS

#### What MonoGame platform are you using:
<!-- e.g. DesktopGL, WindowsDX, WindowsUWP, Android -->
- DesktopGL

--- a/MonoGame.Framework.Content.Pipeline/ExternalTool.cs
+++ b/MonoGame.Framework.Content.Pipeline/ExternalTool.cs
@@ -125,12 +125,12 @@ namespace Microsoft.Xna.Framework.Content.Pipeline
                 return command;
 
             // For Linux check specific subfolder
-            var lincom = "linux/" + command;
+            var lincom = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "linux", command);
             if (CurrentPlatform.OS == OS.Linux && File.Exists(lincom))
                 return lincom;
 
             // For Mac check specific subfolder
-            var maccom = "osx/" + command;
+            var maccom = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "osx", command);
             if (CurrentPlatform.OS == OS.MacOSX && File.Exists(maccom))
                 return maccom;
 

